examples/provider-install-verification/main.tf

```diff
@@ -0,0 +1,41 @@
+terraform {
+  required_providers {
+    meshstack = {
+      source = "meshcloud/meshstack"
+    }
+  }
+}
+
+locals {
+  workspace = "henrys-workspace"
+}
+
+provider "meshstack" {
```

Comment:
probably not critical, but better put secrets here.
guess the whole example can be reworked a bit later.